app/views/casework/crime_applications/_interests_of_justice.html.erb

```diff
@@ -29,15 +29,13 @@
   </table>
 <% else %>
   <dl class="govuk-summary-list govuk-!-margin-bottom-9">
-    <% crime_application.ioj_passport.each do |passport| %>
-      <div class="govuk-summary-list__row">
-        <dt class="govuk-summary-list__key">
-          <%= label_text(:justification) %>
-        </dt>
-        <dd class="govuk-summary-list__value">
-          <%= t(passport, scope: 'values') %>
-        </dd>
-      </div>
-    <% end %>
+    <div class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key">
+        <%= label_text(:justification) %>
+      </dt>
+      <dd class="govuk-summary-list__value">
+        <%= t(crime_application.precedent_ioj_passport, scope: 'values') %>
```

Comment:
Is this ment to be `present_ioj_passport` rather than precedent?

Reply:
Precedent i.e. the most important - I struggled to find the right name. https://en.wiktionary.org/wiki/precedent#:~:text=justify%20a%20judgment-,Adjective,arrangement%3B%20preceding%2C%20foregoing.%20%5B

Wiktionary say that the usage I have gone for is 'now rare'

Reply:
Maybe I was thinking https://www.merriam-webster.com/dictionary/preeminent - would that be any better?? Is there anything you'd suggest?